src/components/LayerUI.tsx

```diff
@@ -211,7 +210,27 @@ const LibraryMenu = ({
   setAppState: React.Component<any, AppState>["setState"];
 }) => {
   const ref = useRef<HTMLDivElement | null>(null);
-  useOnClickOutside(ref, onClickOutside);
+  useOnClickOutside(ref, (ev) => {
+    let target = ev.target;
+    let isClickOnLibraryButton = false;
```

Comment:
- You don't need this variable if you simply `return` instead of `break`
- You don't actually need a loop. Just use `.closest(selector)`

Reply:
Thanks for your comment👍 .  Updated

Reply:
IE doesn't support `.closest(selector)`. Is that matter?

Reply:
It's fine, we don't support IE by any stretch :).

Reply:
Aha 😃